source/User_Guide/Legacy_Newsletter/Migration_Tutorials/migrating_recipient_lists.md

```diff
@@ -8,6 +8,9 @@ title: Migrating Recipient Lists
 navigation:
   show: true
 ---
```

Comment:
Your Legacy Newsletter unsubscribes are not automatically migrated when you move your recipient lists. You will need to [migrate your unsubscribes]

^^ I couldn't comment on line 35, but it was a broken redirect